File: week3/ex2.py
#!/usr/bin/env python3

import sys

from numpy import append
AF_file = open ("AF.txt", "w")
for line in open ("/Users/cmdb/qb25-answers/week3/BYxRM_bam/biallelic.vcf"): 
    if line.startswith('#'): 
        continue
    fields= line.rstrip('\n').split('\t')
    allele_freq = fields[7].split(";")
    for line in allele_freq: 
      if line.startswith('AF='):  
        stripped_AF = line.strip("AF=")
        AF_file.write(stripped_AF + "\n")
        continue 
AF_file.close()
    #print(AF)

File: week3/ex2.R
setwd("/Users/cmdb/qb25-answers/week3")
library (tidyverse)
AF_file <-read_tsv("/Users/cmdb/qb25-answers/week3/AF.txt", col_names = FALSE)
ggplot(data = AF_file, aes(x = X1)) + 
  geom_histogram (bins = 11) + xlab("AF") + ylab ("counts")
ggsave ("AF.png")

setwd("/Users/cmdb/qb25-answers/week3")
library (tidyverse)
AF_file <-read_tsv("/Users/cmdb/qb25-answers/week3/DP.txt", col_names = FALSE)
ggplot(data = AF_file, aes(x = X1)) + 
  geom_histogram (bins = 21) + xlab("DP") + ylab ("counts")+ 
  xlim(0,20)
ggsave ("DP.png")







